src/styles/ItemsList.css

```diff
@@ -0,0 +1,10 @@
+.shoppinglist ul {
+  margin: 0;
+  padding: 0;
+  list-style: none;
+}
+
+input[type='checkbox']:checked + label {
+  opacity: 0.5;
+  text-decoration: line-through;
```

Comment:
Should this CSS change the styling of the label of the item to strikethrough and faded when the item is checked? It didn't seem to change when I ran the code, but I'm not sure why?

Reply:
You are right, I guess I didn't check after I changed the position of label and input! 🙅

Reply:
@saliainred I pushed some changes a few hours ago. Essentially that CSS will work only if the `input` is not nested inside `label`. I guess if you make changes to a branch after opening a PR, the PR will not reflect the latest changes. Check out my latest commit https://github.com/the-collab-lab/tcl-19-smart-shopping-list/pull/24/commits/a06a2de259807bbf261b4875619d12bd820e44f0

Reply:
It seems everything works fine now with the latest changes you made, Joseph :)